recipes/highfive/all/conanfile.py

```diff
@@ -27,7 +27,7 @@ class HighFiveConan(ConanFile):
         "with_boost": True,
         "with_eigen": True,
         "with_xtensor": True,
-        "with_opencv": True,
+        "with_opencv": False,
```

Comment:
I was holding off since we usually dont change these but this matches upstream https://github.com/BlueBrain/HighFive/blob/577a2552357ddd9ba9ed412452ea5406747033b4/CMakeLists.txt#L23

So it's acceptable to me

Reply:
Thanks for looking into the opencv/jaspar deps issue.
I can submit another PR once https://github.com/conan-io/conan-center-index/issues/13159 is solved to set `with_opencv` back to True.